public/app/features/search/loaders.ts

```diff
@@ -35,7 +36,7 @@ export const loadFolderPage = (uid: string, activeChildId: string) => {
     },
   };
 
-  return backendSrv.getFolderByUid(uid).then((folder: any) => {
+  return backendSrv.getFolderByUid(uid).then(folder => {
     const folderTitle = folder.title;
     const folderUrl = folder.url;
     navModel.main.text = folderTitle;
```

Comment:
Can't comment on unchanged code, but I think `navModel.main.children.find` below need to use optional chaining `navModel.main.children?.find` since `main.children` is optional.